ui/src/app/pages/access/two-factor-auth-modal.spec.tsx

```diff
@@ -1,14 +1,21 @@
 import * as React from 'react';
-import { mount } from 'enzyme';
+
+import { render, RenderResult, screen } from '@testing-library/react';
 
 import { TwoFactorAuthModal } from './two-factor-auth-modal';
 
 describe('TwoFactorAuthModal', () => {
-  const component = () =>
-    mount(<TwoFactorAuthModal onCancel={() => {}} onClick={() => {}} />);
+  const component = (): RenderResult =>
+    render(
+      <div id='popup-root'>
+        <TwoFactorAuthModal onCancel={() => {}} onClick={() => {}} />
+      </div>
+    );
 
   it('should render', () => {
-    const wrapper = component();
-    expect(wrapper).toBeTruthy();
+    component();
+    expect(
+      screen.queryByText('Redirecting to turn on Google 2-step Verification')
```

Comment:
Changed test to be different than just component is truthy, because since I wrapped the component in a div, it should always be truthy. Looking for title instead.